### PR TITLE
Added entries for alternate centos7 tags and centos6 container.

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This is a temporary measure until multiple desired-tag support is added.

### CODE REVIEW

File: index.d/centos.yml [hunk 1]
```diff
@@ -330,3 +330,27 @@ Projects:
...
+    depends-on: null
+
+  - id: 32
+    app-id: centos
+    job-id: centos
+    git-url: https://github.com/CentOS/sig-cloud-instance-images
+    git-branch: CentOS-7
+    git-path: docker
+    target-file: Dockerfile
+    desired-tag: 7.2.1511
```
Comment (kbsingh) [inline index.d/centos.yml]: this is the point in time image, its not the same image as the centos:7 one is it ?

Comment (mohammedzee1000) [inline index.d/centos.yml]: Git branch updated.

**Comment by mohammedzee1000** [comment]
We need to get the cccp.yml file into the appropriate branch. I will raise a PR for the same.

**Comment by kbsingh** [comment]
lgtm, work through the tests

**Comment by mohammedzee1000** [comment]
PR added to cccp.yml file https://github.com/CentOS/sig-cloud-instance-images/pull/66

**Comment by mohammedzee1000** [comment]
I think this PR is ready to merge.

File: index.d/centos.yml (hunk 1 above)
Comment (kbsingh) [inline index.d/centos.yml]: you should be able to get the 7.2.1511 content, and add an entry for that as well

**Comment by mohammedzee1000** [comment]
@kbsingh i think we are good here now

**Pull request merged**